x-pack/plugins/security_solution/public/management/components/endpoint_action_failure_message/endpoint_action_failure_message.tsx

```diff
@@ -14,11 +14,19 @@ import type { ActionDetails, MaybeImmutable } from '../../../../common/endpoint/
 
 interface EndpointActionFailureMessageProps {
   action: MaybeImmutable<ActionDetails>;
+  isConsoleOutput?: boolean;
```

Comment:
Suggestion: rename to `showSummaryCount`

`isConsoleOutput` ties this option to console when really its just controlling if we show an additional summary line or not.  Also - why not show this summary line on the Action Log as well?

Reply:
Yeah makes sense to show the summary line as well.